docs/user-guide.md

```diff
@@ -160,13 +197,19 @@ To export all conversations, at the bottom of the left panel, click the **Export
 
 Exported conversations can be imported. 
 
-To import JSON or a zip file with conversations, click the **Import conversations** icon at the bottom of the left panel and then select a file with conversations.
+To import a JSON file with a conversation or a ZIP archive with several conversations (may include attachments as well), click the **Import conversations** icon at the bottom of the left panel and then select a file with conversations.
 
-> When you import a conversation with attachments, the attachments will be available in the Imports folder.
- 
 ![](./img/import_conv.png)
 
-> Note, that when importing a duplicate of an existing conversation, you will be prompted to select one of the proceeding options: replace, ignore or add a prefix to the imported conversation. If the imported conversation includes attachments, you will be prompted to make the same selection for them as well.
+When you import a conversation with attachments, the attachments will be available in the **Imports** folder in the [Attachments Manager](#attachments-manager).
```

Comment:
Sorry, this has been changed. Now there is no specific folder created after import. The attachments will be available in the parent folder or in the root folder. We removed creation of import folder.

Reply:
fixed